### ISSUE TITLE
About random seed

### ISSUE THREAD

**Issue by ImageProcessing01** [ali-chr/Transductive_ZSL_3D_Point_Cloud#5]
Dear Sir,
I ran the provided code and i can not reproduce the result in the paper. I wonder if the problem is in the random seed ? can you give a reproducible version?
I encounter the below situation when i directly run the given code (without any change of hyperpara):
In inductive ZS, I got: 

- Epoch= 1  ZSL: acc= 14.867841409691628 , GZSL: acc_S= 90.32051282051282 , acc_U= 0.0 , HM= 0.0
- Epoch= 2  ZSL: acc= 14.867841409691628 , GZSL: acc_S= 90.51282051282051 , acc_U= 0.0 , HM= 0.0
- Epoch= 3  ZSL: acc= 15.088105726872246 , GZSL: acc_S= 90.0 , acc_U= 0.0 , HM= 0.0
- Epoch= 4  ZSL: acc= 15.198237885462554 , GZSL: acc_S= 90.57692307692308 , acc_U= 0.0 , HM= 0.0
- Epoch= 5  ZSL: acc= 15.418502202643172 , GZSL: acc_S= 90.76923076923077 , acc_U= 0.0 , HM= 0.0

In transductive ZS, I got:

- Epoch= 1  ZSL: acc= 35.02202643171806 , GZSL: acc_S= 89.42307692307693 , acc_U= 10.462555066079295 , HM= 18.733302234850093
- Epoch= 2  ZSL: acc= 34.91189427312775 , GZSL: acc_S= 89.48717948717949 , acc_U= 10.13215859030837 , HM= 18.20325876203327Epoch= 3  ZSL: acc= 36.23348017621145 , GZSL: acc_S= 89.48717948717949 , acc_U= 10.572687224669604 , HM= 18.911077746294435
- Epoch= 4  ZSL: acc= 36.12334801762114 , GZSL: acc_S= 89.35897435897436 , acc_U= 10.13215859030837 , HM= 18.20060286445768Epoch= 5  ZSL: acc= 37.88546255506608 , GZSL: acc_S= 89.42307692307693 , acc_U= 9.361233480176212 , HM= 16.948244071866558

Is there anything wrong with my training?

And thanks for your time.

**Comment by townim-faisal** [comment]
Hi, can you increase the learning rate?
Or you can change the optimizer as:
```
optimizer = optim.Adam(model.parameters() , lr=0.01, betas=(0.9, 0.999),weight_decay=0.0000001)
```
Or in the config files
